get_github_coderabbit_review/coderabbit_fetcher/github_client.py

```
@@ -312,6 +315,19 @@ def get_pr_info(self, pr_url: str) -> Dict[str, Any]:
 
             pr_info = json.loads(result.stdout)
 
+            # Fetch files information separately
+            try:
+                files_result = subprocess.run([
```

Ruff (0.13.1)

`subprocess` call: check for execution of untrusted input

(S603)